### ISSUE TITLE
Add analysis of fully-qualified expression in NewLines rule

### ISSUE THREAD

**Issue by Arrgentum** [saveourtool/diktat#1403]
## Describe the bug

Fully-qualified expression should not be separated

## Initial code:

```kotlin
private val holder: java.util.concurrent.atomic.AtomicReference<T> = java.util.concurrent.atomic.AtomicReference(valueToStore)
```

```kotlin
private val holder: kotlin.native.concurrent.AtomicReference<T> = kotlin.native.concurrent.AtomicReference(valueToStore)
```

## Expected behavior

```kotlin
private val holder: java.util.concurrent.atomic.AtomicReference<T> = java.util.concurrent.atomic.AtomicReference(valueToStore)
```

```kotlin
private val holder: kotlin.native.concurrent.AtomicReference<T> = kotlin.native.concurrent.AtomicReference(valueToStore)
```

## Observed behavior

```kotlin
private val holder: java.util.concurrent.atomic.AtomicReference<T> = java.util
        .concurrent
        .atomic
        .AtomicReference(valueToStore)
```

```kotlin
private val holder: kotlin.native.concurrent.AtomicReference<T> = kotlin.native
        .concurrent
        .AtomicReference(valueToStore)
```

## Environment information
* diktat version: 1.2.0
* kotlin version: 1.6

**Comment by akuleshov7** [comment]
Solution: `parentExpressionList?.dropWhile { !it.treeParent.textContains('(') && !it.treeParent.textContains('{') }`


That will format the following: `java.util.Test().map().size`

To the following format:
```
java.util.Test()
.map()
.size
```

Only the following code won't be covered: 
```
test.list.size
```

but that is the problem of code author :)
